core/tag_api.php

```diff
@@ -435,36 +466,78 @@ function tag_get_all( $p_name_filter, $p_count, $p_offset ) {
 	}
 
 	$t_query = 'SELECT * FROM {tag} ' . $t_where . ' ORDER BY name';
+	$t_query = <<<SQL
```

Comment:
Above assignment to `$t_query` can be removed.

Reply:
Indeed. Thanks for catching it.

Reply:
Fixed in e240d75447a222c80c6918b7e322b958a7208ae0